openshift/kustomize/services/extract-quotes/overlays/dev/config-map.yaml

```diff
@@ -19,3 +19,9 @@ data:
   TOPICS: index
   CHES_EMAIL_ENABLED: "true"
   CHES_EMAIL_AUTHORIZED: "true"
+  USE_LLM: "true"
```

Comment:
Often for overlays you won't have a new config-map.yaml, but rather one that overrides the values in the base.  It isn't critical to do this.